From: Moni Shoua <monis@mellanox.com>
Date: Thu, 14 Jan 2016 17:50:38 +0200
Subject: IB/core: Add definition for the standard RoCE V2 UDP port
Patch-mainline: v4.5-rc1
Git-commit: 7ead4bcb1b788732516755ef84ef1272d3e152eb
References: bsc#966191 FATE#320230 bsc#966186 FATE#320228

This will be used in hardware device driver when building QP or AH
contexts.

Signed-off-by: Moni Shoua <monis@mellanox.com>
Signed-off-by: Matan Barak <matanb@mellanox.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 include/rdma/ib_verbs.h |    1 +
 1 file changed, 1 insertion(+)

--- a/include/rdma/ib_verbs.h
+++ b/include/rdma/ib_verbs.h
@@ -77,6 +77,7 @@ enum ib_gid_type {
 	IB_GID_TYPE_SIZE
 };
 
+#define ROCE_V2_UDP_DPORT      4791
 struct ib_gid_attr {
 	enum ib_gid_type	gid_type;
 	struct net_device	*ndev;
